sdk/core/core-rest-pipeline/src/fetchHttpClient.ts

```diff
@@ -75,7 +75,6 @@ async function makeRequest(request: PipelineRequest): Promise<PipelineResponse>
       headers: headers,
       signal: abortController.signal,
       credentials: request.withCredentials ? "include" : "same-origin",
-      redirect: "manual",
```

Comment:
Allowing Fetch to automatically handle redirects means that our `redirectPolicies` will never kick in for browsers right?

Reply:
If that's the behavior we want, I think it would be worth adding a comment in the redirect policy mentioning this, it would save future us a headache trying to debug redirects and realize later that Fetch is handling it for us

Reply:
Yeah, the policy won't kick in, but it won't kick in now since we get back `status: 0` and no headers. I'll add a comment here too.